src/controllers/Talk_commentsController.php

```diff
@@ -130,4 +130,47 @@ public function moderateReportedComment($request, $db)
         header("Location: $uri", true, 204);
         exit;
     }
+
+    public function updateComment($request, $db)
+    {
+        $request->user_id = 14;
```

Comment:
@runz0rd - I guess this was put in for testing? Needs removing before I can merge please :-)